latest/index.bs

```diff
@@ -141,19 +141,23 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
     │                         # "multiscales" and "omero" (see below). In addition, the group level attributes
     │                         # must also contain "_ARRAY_DIMENSIONS" if this group directly contains multi-scale arrays.
     │
-    ├── 0                     # Each multiscale level is stored as a separate Zarr array,
-    │   ...                   # which is a folder containing chunk files which compose the array.
-    ├── n                     # The name of the array is arbitrary with the ordering defined by
+    ├── 0                     # Each multiscale level is stored as a separate Zarr group
```

Comment:
-> *separate Zarr group, possibly nested* ?

Reply:
Happy to add.